### PR TITLE
Set myself as proxied maintainer for nudoku and profanity

### PR DESCRIPTION
**Pull request by jubalh**
I'm upstream for both packages and would like to set myself as maintainer already to receive all the bugs even when so far I didn't do any changes to the ebuild.

### CODE REVIEW

**Comment by jubalh** [comment]
I'm upstream for both packages and would like to set myself as maintainer already to receive all the bugs even when so far I didn't do any changes to the ebuild.

**Comment by a17r** [comment]
@gentoo/proxy-maint existing maintainer bug: https://bugs.gentoo.org/647718

File: games-puzzle/nudoku/metadata.xml [hunk 1]
```diff
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+		<maintainer type="person" proxied="yes">
```
Comment (arkamar) [inline games-puzzle/nudoku/metadata.xml:4]: align indetation of this block, one tab left.

File: net-im/profanity/metadata.xml [hunk 2]
```diff
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+		<maintainer type="person" proxied="yes">
```
Comment (arkamar) [inline net-im/profanity/metadata.xml:4]: The same here.

**Review by arkamar — CHANGES_REQUESTED** [review]

File: games-puzzle/nudoku/metadata.xml (hunk 1 above)
Comment (jubalh) [reply games-puzzle/nudoku/metadata.xml:4]: My bad! I changed that and pushed. but seems already merged?

Comment (Flowdalic) [reply games-puzzle/nudoku/metadata.xml:4]: I fixed those while merging it. But certainly something you may want to look at in the future. :)

Comment (jubalh) [reply games-puzzle/nudoku/metadata.xml:4]: Will do! Thanks

**Pull request closed without merging**